src/database/DataTypes/Stocktake.js

```diff
@@ -250,7 +250,8 @@ export class Stocktake extends Realm.Object {
 
   get canFinalise() {
     const finaliseStatus = { success: true, message: modalStrings.finalise_stocktake };
-    if (!this.hasSomeCountedItems || !this.hasValidDoses) {
+
+    if (!this.hasSomeCountedItems) {
```

Comment:
I have removed the check for valid doses as it was introduced initially when I was restricting the doses value to be within `totalQuantity >= doses <= totalQuantity * item.doses` - now the only restriction is `doses <= totalQuantity * item.doses`, so I'm not sure where to go with this. Could change the validity to be just that `doses <= totalQuantity * item.doses`, but also it is a pretty bad UI experience to go to finalise only to be told no and then have to find the batch. Maybe that's a better solution though. I will make an issue for this as it has to change either way!

Reply:
would be nice to have the validation earlier, and some sort of notification to the user. I'm currently typing in a number for `DOSES` in a CI and it is just changing back to `0` for some reason

Reply:
Do you have 0 quantity for that item? That happens for normal quantity too :( https://github.com/openmsupply/mobile/issues/1949 unless it's something different, of course

Reply:
yes, it is a zero stock issue. I'm having a separate problem, in that I cannot finalise a stocktake with a vaccine in it, so I couldn't quickly give myself stock